cogs5e/initiative/cog.py

```diff
@@ -618,7 +619,8 @@ async def metasetting(self, ctx, *settings):
         `dyn` - Dynamic initiative; Rerolls all initiatves at the start of a round.
         `turnnotif` - Notifies the controller of the next combatant in initiative.
         `deathdelete` - Toggles removing monsters below 0 HP.
-        `-name <name>` - Sets a name for the combat instance
+        `-name <name>` - Sets a name for the combat instance.
+        `-combatdm <@mention>` - Changes this combat's dm. You must be a DM or the combat's current DM.
```

Comment:
```suggestion
        `-combatdm <@mention>` - Changes this combat's DM.
```
Since changing the DM is an explicit action, I think we shouldn't add this restriction (in case the old DM goes AFK or something).